examples/tensorize_vllm_model.py

```diff
@@ -0,0 +1,230 @@
+import argparse
+import os
+import tempfile
+import time
+import uuid
+from functools import partial
+from typing import Type
+
+import torch
+import torch.nn as nn
+from tensorizer import (DecryptionParams, EncryptionParams, TensorDeserializer,
+                        TensorSerializer, stream_io)
+from tensorizer.utils import convert_bytes, get_mem_usage, no_init_or_tensor
+from transformers import AutoConfig, AutoModelForCausalLM, PretrainedConfig
+
+from vllm.config import _get_and_verify_dtype
+from vllm.model_executor.models import ModelRegistry
+from vllm.model_executor.parallel_utils.parallel_state import (
+    initialize_model_parallel)
+from vllm.model_executor.tensorizer_loader import TensorizerArgs
+
+
+def parse_args():
```

Comment:
So in order to use the tensorizer, we should serialize the model using this script and store it in S3 or local file?  Is it possible to add e2e example somewhere? (like I can follow line by line)? I think it is important to make serializing the model part straightforward and easy.

Reply:
Yep Let me know if this recent docstring addition answers your question :)

https://github.com/vllm-project/vllm/pull/3476/commits/71a9f795b865b296c9ce34b1e3b669fa8bc36580#diff-b1567039ae60823bd62b460014416422145b1cf46be70d7265c66af6cfcaf974R22-R55

Reply:
How do you usually set the credentials for S3? Is it a part of the API? Can you also add a short docstring to explain it?